scripts/globals/garrison_data.lua

```diff
@@ -8,106 +8,30 @@ xi = xi or {}
 xi.garrison = xi.garrison or {}
 
 -- Name is Determined by Nation and LevelCap
-xi.garrison.names =
+-- Names in order of xi.nation values (sandoria, bastok, windurst)
+xi.garrison.allyNames =
 {
-    --level 20 garrison names
-    [20] =
-    {
-        [0] =
-        {
-            npcName = "Patrician"
-        },
-        [1] =
-        {
-            npcName = "Recruit"
-        },
-        [2] =
-        {
-            npcName = "Candidate"
-        },
-    },
-    --level 30 garrison names
-    [30] =
-    {
-        [0] =
-        {
-            npcName = "Trader"
-        },
-        [1] =
-        {
-            npcName = "Mariner"
-        },
-        [2] =
-        {
-            npcName = "Scholar"
-        },
-    },
-    --level 40 garrison names
-    [40] =
-    {
-        [0] =
-        {
-            npcName = "TempleKnight"
-        },
-        [1] =
-        {
-            npcName = "GoldMusketeer"
-        },
-        [2] =
-        {
-            npcName = "UNKNOWN"
-        },
-    },
-    --level 50 garrison names
-    [50] =
-    {
-        [0] =
-        {
-            npcName = "RoyalGuard"
-        },
-        [1] =
-        {
-            npcName = "GoldMusketeer"
-        },
-        [2] =
-        {
-            npcName = "Patriarch"
-        },
-    },
-    --level 99 garrison names
-    [75] =
-    {
-        [0] =
-        {
-            npcName = "MilitaryAttache"
-        },
-        [1] =
-        {
-            npcName = "MilitaryAttache"
-        },
-        [2] =
-        {
-            npcName = "MilitaryAttache"
-        },
-    },
+    [20] = { "Patrician", "Recruit", "Candidate" },
+    [30] = { "Trader", "Mariner", "Scholar" },
+    [40] = { "TempleKnight", "GoldMusketeer", "UNKNOWN" },
```

Comment:
I know its a pain in the ass, but you can't check in UNKNOWN as a name that players will see

Reply:
Changed to TempleKnight and added a TODO. Will be fixing some of these later today.